dashboard/dashapi/dashapi.go

```diff
@@ -17,6 +17,8 @@ import (
 	"reflect"
 	"strings"
 	"time"
+
+	"github.com/google/syzkaller/pkg/vcs"
```

Comment:
We did not import vcs here before, nor any other packages, due to the backward/forward compatibility concerns.
It's easier to reuse the vcs.Recipients type here, but then an innocently looking change local to vcs package can break syzbot deployment. That will be easy to overlook. It also won't be possible to change/fix anything in vcs.Recipients declaration later.
Please duplicate the type here.